Delete resize_dim()

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32116 Refreshing numel on a stride update is pointless.
* **#32114 Delete resize_dim()**
* #32112 Expunge last two sites of resize_dim

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D19579876](https://our.internmc.facebook.com/intern/diff/D19579876)

ezyang committed:
Delete resize_dim()

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 40336d38:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=40336d3841bb37f27de0863f52c7cbb33f70d186).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4222030) pytorch_xla_linux_xenial_py3_6_clang7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=84386942) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=84386942))
<details>
<summary>
<code>Jan 13 20:56:30 Failed to generate ATEN bindings: ['/var/lib/jenkins/workspace/xla/scripts/generate_code.sh']</code>
</summary>

```
Jan 13 20:56:29 Generated 877 wrappers for /var/lib/jenkins/workspace/xla/scripts/../../torch/csrc/autograd/generated/RegistrationDeclarations.h 
Jan 13 20:56:29 AtenXlaType function missed override: Tensor& bitwise_or_out(Tensor& out, const Tensor& self, const Tensor& other); // bitwise_or_out(Tensor,Tensor,Tensor)->Tensor 
Jan 13 20:56:29 AtenXlaType function missed override: Tensor& bitwise_or_out(Tensor& out, const Tensor& self, Scalar other); // bitwise_or_out(Tensor,Tensor,Scalar)->Tensor 
Jan 13 20:56:29 Traceback (most recent call last): 
Jan 13 20:56:29   File "/var/lib/jenkins/workspace/xla/scripts/gen.py", line 1052, in <module> 
Jan 13 20:56:29     generate(args) 
Jan 13 20:56:29   File "/var/lib/jenkins/workspace/xla/scripts/gen.py", line 1022, in generate 
Jan 13 20:56:29     assert check_overrides(overrides, overridden) 
Jan 13 20:56:29 AssertionError 
Jan 13 20:56:30 Building torch_xla version: 0.8 
Jan 13 20:56:30 Failed to generate ATEN bindings: ['/var/lib/jenkins/workspace/xla/scripts/generate_code.sh'] 
Jan 13 20:56:30 =================== sccache compilation log =================== 
Jan 13 20:56:30 + cleanup 
Jan 13 20:56:30 + retcode=1 
Jan 13 20:56:30 + set +x 
Jan 13 20:56:30 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 13 20:56:30 Compile requests               2525 
Jan 13 20:56:30 Compile requests executed      2268 
Jan 13 20:56:30 Cache hits                     1249 
Jan 13 20:56:30 Cache misses                   1007 
Jan 13 20:56:30 Cache timeouts                    0 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ezyang committed:
Update on "Delete resize_dim()"


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Delete resize_dim()"


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Delete resize_dim()"


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Delete resize_dim()"


Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D19579876](https://our.internmc.facebook.com/intern/diff/D19579876)

[ghstack-poisoned]
